lib/notification-issue.coffee

```diff
@@ -55,7 +55,13 @@ class NotificationIssue
 
   getIssueTitle: ->
     title = @notification.getMessage()
-    title = title.replace(process.env.ATOM_HOME, '$ATOM_HOME')
+    if process.platform is 'win32'
+      title = title.replace(process.env.ATOM_HOME, '%ATOM_HOME%')
```

Comment:
The text that will show up in the issue title should still be `$ATOM_HOME` so that cross-platform issues still match.